runtime/lua/vim/fs.lua

```diff
@@ -76,14 +76,28 @@ end
 --- The search can be narrowed to find only files or or only directories by
 --- specifying {type} to be "file" or "directory", respectively.
 ---
----@param names (string|table|fun(name: string): boolean) Names of the files
+---  Examples: >
+---     
+---    vim.fs.find("aFileOrDirectoryNextToNeovim")
+---    => "C:/Users/jdoe/aFileOrDirectoryNextToNeovim"
+---
+---    vim.fs.find("allFilesOrDirectoriesAboveNeovim", {upward = true, limit = math.huge}
+---    => "/home/jdoe/allFilesOrDirectoriesAboveNeovim"
+---    => "/home/allFilesOrDirectoriesAboveNeovim"
+---       
+---    vim.fs.find("aFileInHomeDirectory", {path = "/home"})
+---    => "/home/jdoe/archive/aFileInHomeDirectory"
+--- <
+---
+---@param names (string or table or fun(name: string): boolean) Names of the files
```

Comment:
The format this was in before was intentional. Please don't make arbitrary changes without first understanding why it was done a certain way in the first place.

```suggestion
@param names (string|table|fun(name: string): boolean) Names of the files
```